furious/async.py

```diff
@@ -91,6 +91,7 @@ def run_me(*args, **kwargs):
 MAX_DEPTH = 100
 MAX_RESTARTS = 10
 DISABLE_RECURSION_CHECK = -1
+RETRY_SLEEP_SECS = 4
```

Comment:
You could just make the retry option take the number of seconds to sleep and use a negative value to indicate no retry. Might make the api a little unintuitive though.